From 9f84f8afd986950dde0f496ce66a6f53dca985cf Mon Sep 17 00:00:00 2001
From: Xiao-Long Chen <chenxiaolong@cxl.epac.to>
Date: Sun, 10 Nov 2013 15:01:19 -0500
Subject: [PATCH] Enable translucent lockscreen decor

Taken from Ethan Chen's commit hash: 0ba9dbb89160cd0fa25c88b6bba0f2b9bbfcd6b9

Change-Id: Ic51f01416d7474278dcd3036306b2c82ce2e74d4
---
 overlay/frameworks/base/core/res/res/values/config.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/overlay/frameworks/base/core/res/res/values/config.xml b/overlay/frameworks/base/core/res/res/values/config.xml
index 138dd0e..0ff02c1 100644
--- a/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/frameworks/base/core/res/res/values/config.xml
@@ -250,4 +250,7 @@
 
     <!-- Device supports LED flashlight -->
     <bool name="config_enableTorch">true</bool>
+
+    <!-- Disable lockscreen translucent decor by default -->
+    <bool name="config_enableLockScreenTranslucentDecor">true</bool>
 </resources>
-- 
1.8.4.2

